scripts.d/25-fftw3.sh

```diff
@@ -29,11 +29,16 @@ ffbuild_dockerbuild() {
         --enable-threads
         --with-combined-threads
         --with-incoming-stack-boundary=2
-        --enable-sse2
-        --enable-avx
-        --enable-avx2
     )
 
+    if [[ $TARGET != linuxarm64 ]]; then
```

Comment:
Would prefer those checks to be testing *arm64, since winarm64 is a distinct possibility, whenever mingw/crosstools gets it together to build a toolchain.